### PR TITLE
3190: Code and Text cells from tool bar

### PR DESCRIPTION
**Pull request by rajmusuku**
- Added buttons to tool bar. Need to add related images.
- Hooked up with 'addCell' event to create cell base on cell type

### CODE REVIEW

**Comment by kevcunnane** [comment]
Minor: It's nice to add screenshots when adding new UI components so we can review how it looks too. THanks!

File: src/sql/parts/notebook/notebook.css [hunk 1]
```diff
@@ -23,4 +23,17 @@
 	text-align: center;
 	display: flex;
 	align-items: center;
+}
+
+.notebookEditor .notebook-info-button {
```
Comment (kevcunnane) [inline src/sql/parts/notebook/notebook.css]: How does this compare to our other buttons (e.g. in the SQL editor toolbar, or the ModelViewEditor one)?

**Comment by rajmusuku** [comment]
Please see screenshot attached for the buttons added on toobar.
image

File: src/sql/parts/notebook/notebook.css (hunk 1 above)
Comment (rajmusuku) [reply src/sql/parts/notebook/notebook.css]: Extracted some CSS components from Aris extension. We may have to revisit the CSS again. Did not go through SQL toolbar.

File: src/sql/parts/notebook/media/light/add.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><title>add_16x16</title><path d="M6,1a5.17,5.17,0,0,1,1.4.19,5.26,5.26,0,0,1,3.67,3.67,5.3,5.3,0,0,1,0,2.79A5.26,5.26,0,0,1,7.4,11.32a5.3,5.3,0,0,1-2.79,0A5.26,5.26,0,0,1,.94,7.65a5.3,5.3,0,0,1,0-2.79A5.26,5.26,0,0,1,4.6,1.19,5.17,5.17,0,0,1,6,1Zm0,9.75a4.4,4.4,0,0,0,1.2-.16,4.56,4.56,0,0,0,1.08-.45A4.5,4.5,0,0,0,9.88,8.53a4.56,4.56,0,0,0,.45-1.08,4.51,4.51,0,0,0,0-2.39A4.56,4.56,0,0,0,9.88,4,4.5,4.5,0,0,0,8.27,2.37a4.56,4.56,0,0,0-1.08-.45,4.51,4.51,0,0,0-2.39,0,4.56,4.56,0,0,0-1.08.45A4.5,4.5,0,0,0,2.11,4a4.56,4.56,0,0,0-.45,1.08,4.51,4.51,0,0,0,0,2.39,4.56,4.56,0,0,0,.45,1.08,4.5,4.5,0,0,0,1.61,1.61,4.56,4.56,0,0,0,1.08.45A4.4,4.4,0,0,0,6,10.76Zm.38-4.88H8.25v.75H6.37V8.51H5.62V6.63H3.75V5.88H5.62V4h.75Z"/></svg>
```
Comment (chlafreniere) [inline src/sql/parts/notebook/media/light/add.svg]: What about the dark theme version of these icons?

Comment (rajmusuku) [reply src/sql/parts/notebook/media/light/add.svg]: That needs a little bit of discussion to port over them.

**Pull request merged**